content/support-matrix/_index.md

```diff
@@ -0,0 +1,10 @@
+---
+title: Support Matrix
+description: "All supported matrix data for SODA can be found here"
```

Comment:
All supported matrix data for SODA? 
This statement does not give the intended meaning.

Reply:
corrected the statements